src/general/StageHUDBase.cs

```diff
@@ -529,7 +498,13 @@ public override void _Process(float delta)
 
         UpdateATP(delta);
         UpdateHealth(delta);
-        UpdateHoverInfo(delta);
+
+        hoverInfoTimeElapsed += delta;
+        if (hoverInfoTimeElapsed > Constants.HOVER_PANEL_UPDATE_INTERVAL)
+        {
+            UpdateHoverInfo(delta);
```

Comment:
This should pass `hoverInfoTimeElapsed` as this is time limited so the delta is not really the delta of when this method was last called.